my-app/src/HW_37/BurgerMenu.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import style from "./BurgerMenu.module.css";
+
+type Props = {
+  enabled: boolean;
+};
+
+type State = {
+  opened: boolean;
+};
+
+export class ButtonComponent extends React.Component<Props, State> {
```

Comment:
names of file and export should be the same

Reply:
Ok , I get it